src/runtime/graph_executor/debug/graph_executor_debug.cc

```diff
@@ -207,6 +200,42 @@ class GraphExecutorDebug : public GraphExecutor {
     return -1;
   }
 
+  /*!
+   * \brief Get number of outputs of the node.
+   * \param index The index of the node.
+   * \return The number of outputs.
+   */
+  size_t NodeGetNumOutputs(size_t index) {
```

Comment:
forgotten function during refactoring? should be removed?

Reply:
removed it. thanks!